### ISSUE TITLE
ModuleNotFoundError: No module named 'einops'

### ISSUE THREAD

**Issue by deltabytesyt** [neonbjb/tortoise-tts#393]
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-13-bdc871981923>](https://localhost:8080/#) in <cell line: 5>()
      3 voice = 'tom'
      4 
----> 5 from einops import rearrange, repeat
      6 from tortoise.api import TextToSpeech
      7 from tortoise.utils.audio import load_audio, load_voice, load_voices

ModuleNotFoundError: No module named 'einops'

**Comment by rohanblkbox** [comment]
Hi , getting same issue .

**Comment by kylegrover** [comment]
> Colab
> The original colab no longer works by a combination of Google's tendency to forward-break things and Python's package management system. I do not intend to keep fixing it so it has been removed. Apologies!
follow the new guide or use colab fixes in other issues/prs

**Comment by francogrex** [comment]
> ... Requirements seem borked and do not work out of the box ...

Indeed, it's a shame. Thanks for your answer it's the only useful one.

**Comment by rachelcenter** [comment]
> Hi , getting same issue .

same. it came out of nowehre and it keeps asking me to install another module, and another and another, when i had all of them installed and had been using them fine for 3-4 days now
